workshop/4-Messaging/4.1-Pinpoint.ipynb

```diff
@@ -442,28 +463,56 @@
    "source": [
     "### Load Abandoned Cart Email Templates\n",
     "\n",
-    "Next we'll create an email template that includes messaging and a coupon code for users who add items to their cart but fail to complete an order."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "with open('pinpoint-templates/abandoned-cart-email-template.html', 'r') as html_file:\n",
-    "    html_template = html_file.read()\n",
-    "    \n",
```

Comment:
Same as above. You still have to load the template source so it can be set in Pinpoint.

Reply:
Good point. Added the template load and links